README.md

```diff
@@ -1,167 +1,211 @@
 
-TITLE requirements:
-	* MAX CHAR: 64 
-	* Industry use case, ie: Protect patient data with LLM guardrails
+## Overview
 
-TITLE will be extracted for publication.
+Imagine we run a successful lemonade stand and want to deploy a customer service agent so our customers can learn more about our products. We'll want to make sure all conversations with the agent are family friendly, and that it does not promote our rival fruit juice vendors.
 
--- > 
+This demo showcases how to deploy an AI-powered customer service assistant with multiple guardrails to ensure safe, compliant, and on-brand interactions. The solution uses Llama 3.2 as the base language model, protected by three detector models that monitor for harmful content, prompt injection attacks, and language compliance.
 
+**In this demo, we are following these assumptions of principles:** 
 
+1. The LLM is untrusted. All its output must be validated. 
+2. The user is untrusted. All the input must be validated.
+3. Triggering of specific detectors are monitored and visualized. (Alerts are out of scope but could be done)
 
-<!-- CONTRIBUTOR TODO: short description 
 
-*ADD a SHORT DESCRIPTION of your use case between H1 title and next section*
+![architecture.png](./docs/images/architecture.png)
 
-SHORT DESCRIPTION requirements:
-	* MAX CHAR: 160
-	* Describe the INDUSTRY use case 
-
-SHORT DESCRIPTION will be extracted for publication.
-
---> 
-
-
-## Table of contents
-
-<!-- Table of contents is optional, but recommended. 
-
-REMEMBER: to remove this section if you don't use a TOC.
+## Detailed description
 
--->
+The Lemonade Stand Assistant provides an interactive customer service experience for a fictional lemonade stand business. Customers can ask questions about products, ingredients, pricing, and more through a conversational interface.
 
-## Detailed description
+To ensure safe and appropriate interactions, the system employs multiple AI guardrails:
+- **IBM HAP Detector (Granite Guardian)**: Monitors conversations for hate, abuse, and profanity
+- **Prompt Injection Detector (DeBERTa v3)**: Identifies and blocks attempts to manipulate the AI assistant
+- **Language Detector (XLM-RoBERTa)**: Ensures inputs and responses are in English only
 
-<!-- CONTRIBUTOR TODO: add detailed description.
+Furthemore, there is a:
+- **Regex Detector**: Blocks specific text without the use of models. In our case, its other fruits we consider "competitors".
 
-This section is required. Describe the quickstart use case in more detail. 
+The guardrails orchestrator coordinates these detectors to evaluate inputs and outputs before presenting responses to users.
 
-This is not a technical description. This is about the workload. 
+### See it in action
 
-Technical description comes later.
+**[▶️ View Interactive Demo](https://demo.arcade.software/X3orbmpyKdY295116jnY)**
```

Comment:
In the arcade, after trying to hijack using turkish there is no a link or a popup that allows me to go to the Grafana dashboard, and I need to do it manually.

Reply:
hihi I love how you recognized Turkish language :D